src/main/java/bit/javaoop/FixedEmployeeNumberManager.java

```diff
@@ -0,0 +1,39 @@
+package bit.javaoop;
+
+public class FixedEmployeeNumberManager extends AbstractManager {
+    private final String position ="manager";
+    Integer limitEmployees = 10;
+    public FixedEmployeeNumberManager(String name, Float salary)
+    {
+        super(name,salary);
+        getEmployeeList().add(this);
+    }
+
+    @Override
+    public String getPosition() {
+        return position;
+    }
+
+    @Override
+    public boolean canHire() {
+        return limitEmployees > getAmountOfEmployees();
+    }
+
+    public void addEmployee(AbstractEmployee employee)
+    {
+        if(canHire() && !getEmployeeList().contains(employee))
+        {
+            super.addEmployee(employee);
+        }
+        else
+        {
+            System.out.printf("Cannot hire a new employee");
```

Comment:
It would probably be better to do something else and by something else I mean one of those two:
1) throw an exception
2) return something to indicate that an employee could not be added